### ISSUE TITLE
cannnot reset to make new file

### ISSUE THREAD

**Issue by msanjulove** [microsoft/vscode#196222]
Type: <b>Feature Request</b>

i need to make new file. the new file saves as a text file even though i click 'new file' and not 'new text file' and i have not discovered the issue.

VS Code version: Code 1.83.1 (Universal) (f1b07bd25dfad64b0167beb15359ae573aecd2cc, 2023-10-10T23:46:55.789Z)
OS version: Darwin x64 22.6.0
Modes:

**Comment by VSCodeTriageBot** [comment]
Thanks for reporting this issue! Unfortunately, it's hard for us to understand what issue you're seeing. Please help us out by providing a screen recording showing exactly what isn't working as expected. While we can work with most standard formats, `.gif` files are preferred as they are displayed inline on GitHub. You may find https://gifcap.dev helpful as a browser-based gif recording tool.

If the issue depends on keyboard input, you can help us by enabling screencast mode for the recording (`Developer: Toggle Screencast Mode` in the command palette). Lastly, please attach this file via the GitHub web interface as emailed responses will strip files out from the issue.

Happy coding!

**Comment by msanjulove** [comment]
Hi! Thanks for your patience!

When I went to create a ‘New File’ the result I continued to get was that of a ‘New Text File’; a blank/black text box would open instead of the larger VS Code work area with the vertically lined numbers on the left of the space. 
I am using VS Code to input HTML into the work area but the text box kept popping up instead of VS Code opening a new untitled file. 

This was unusual as it had opened perfectly before and I have not done anything different in any way to open a new file on VS Code. 

I am unsure as to how to prevent this or recognize a solution to preventing it. I am new to this.
